internal/faustjs.org/docs/gutenberg/getting-started.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: getting-started
+title: Getting Started with Gutenberg Blocks Provider and Viewer
+---
+
+# Getting Started with Gutenberg Blocks Provider and Viewer
+
+## Quick Start
+
+:::note
+
+Make sure you have completed the initial setup for Faust at [Getting Started](/docs/getting-started).
+
+:::
+
+Install the blocks:
+
+```bash
+npm i @faustwp/blocks
+```
+
+Open ```_app.js``` and import the blocks provider:
+
+```js
+import { WordPressBlocksProvider } from '@faustwp/blocks';
+```
+
+Then, inside your templates you need to pass on the ```contentBlocks``` data in your ```WordPressBlocksViewer```:
+
+```js
+import { WordPressBlocksViewer } from '@faustwp/blocks';
+
+const { title, content, featuredImage, date, author, contentBlocks } =
+    props.data.post;
+const blocks = flatListToHierarchical(contentBlocks);
+
+<ContentWrapper>
```

Comment:
You can also remove the `ContentWrapper` as there is no reference to the imports.